scripts/generateAssetData/cosmos/index.ts

```diff
@@ -0,0 +1,96 @@
+import { cosmosChainId } from '@shapeshiftoss/caip'
+import axios from 'axios'
+import type { Asset } from 'lib/asset-service'
+
+import { atom } from '../baseAssets'
+import { colorMap } from '../colorMap'
+import type { IdenticonOptions } from '../generateAssetIcon/generateAssetIcon'
+import { getRenderedIdenticonBase64 } from '../generateAssetIcon/generateAssetIcon'
+
+type CosmosAsset = {
+  denom: string
+  type: string
+  origin_chain: string
+  origin_denom: string
+  origin_type: string
+  symbol: string
+  decimals: number
+  enable?: boolean
+  path?: string
+  channel?: string
+  port?: string
+  counter_party?: {
+    channel: string
+    port: string
+    denom: string
+  }
+  description?: string
+  image?: string
+  coinGeckoId: string
+}
+
+export const getAssets = async (): Promise<Asset[]> => {
+  /* Fetch asset list */
+
+  const { data: assetData } = await axios.get<CosmosAsset[]>(
+    'https://rawcdn.githack.com/cosmostation/chainlist/main/chain/cosmos/assets.json',
+  )
+
+  if (!assetData) throw new Error('Could not get Cosmos asset data!')
+
+  return assetData.reduce<Promise<Asset[]>>(async (accPrevious, current) => {
+    const acc = await accPrevious
+    if (!current) return acc
+
+    const precision = current.decimals
+
+    // IBC assets are all we are interested in here
+    if (!current.denom.startsWith('ibc')) return acc
+    const { assetNamespace, assetReference } = (() => {
+      return { assetNamespace: 'ibc' as const, assetReference: current.denom.split('/')[1] }
+    })()
+
+    // if an asset has a path, it's bridged, so label it as ATOM on Cosmos
+    // Note, IBC assets don't have a name per se and Mintscan display them as "IBC Token" but we may be able to get it by introspecting the origin_denom?
```

Comment:
feels like a bit of a verbose comment for `mark ibc assets with "on Cosmos"`, symbol seems correct, wouldn't do `origin_denom`

Reply:
Actually yeah I've iterated on the original comment but it's still fluff, removed it - `    // IBC assets are all we are interested in here` a few lines above already explains why we're filtering like this, and symbol is self-explanatory